### DISCUSSION TITLE
Feature Request - get agent data from Wikidata

### DISCUSSION THREAD

**Discussion by Jegelewicz** [ArctosDB/arctos#8050] [General]
Issue Documentation is http://handbook.arctosdb.org/how_to/How-to-Use-Issues-in-Arctos.html

**Is your feature request related to a problem? Please describe.**
_A clear and concise description of what the problem is. Ex. I'm always frustrated when [...]_

I often add Wikidata identifiers to agents in Arctos, then I have to manually copy over birth and death dates and any other pertinent information. In yesterday's Bionomia webinar I discovered that Specify can import data from Wikidata, ORCiD and Library of Congress - we should be able to match that.

![image](https://github.com/ArctosDB/arctos/assets/5725767/dabdd550-bc44-40a1-bb74-8acd9c45931e)

**Describe what you're trying to accomplish** 
_A clear and concise overview of the goals; why are you asking for this?_

Less manual labor in the age of AI

**Describe the solution you'd like**
_How might we accomplish your goals?_

If I add a Wikidata address to an agent, Arctos will pull at least:

birth date
death date
alternate names

WIKIDATA REST API
● New way to access and edit data of an
individual entity
● Example:
https://www.wikidata.org/w/rest.php/wi
kibase/v0/entities/items/Q1
● Documentation: Wikidata:REST AP

**Describe alternatives you've considered**
_A clear and concise description of any alternative solutions or features you've considered._

Keep copy pasting

**Additional context**
_Add any other context or screenshots about the feature request here._

**Priority**
_Please assign a priority-label. Unprioritized issues gets sent into a black hole of despair._

**Comment by dustymc** [comment]
Not sure there's anything to discuss here - seems like a good idea, probably technically trivial, ideally would be part of doing the same thing to ORCID and LoC and etc., but also doesn't seem like anything that could be justified for prioritization under current resources. Move to discussions/wishlist?

@mkoo thoughts on prioritizing (funding, whatever) this sort of thing?
